wiki/en/Unregistered-Servers.md

```diff
@@ -29,7 +29,7 @@ You should connect yourself using the local network address of the computer the
 
 ### Dynamic DNS and why you will probably need it
 
-Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a static IP address that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
+Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a domain name that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
```

Comment:
I haven't done this (in a very long time). The dynamic DNS provider gives you an IP address that maps to another IP address? They don't just give you a URL that changes its IP as yours changes?

Reply:
> The dynamic DNS provider gives you an IP address that maps to another IP address

No. It gives you a (sub) domain.

Reply:
Yeah, a DDNS "provider" is just a hostname registrar.  They'll have a bunch of top level domains, like "is-really-cool.com" or whatever.  You buy the subdomain name, e.g. "ice-cream".  Every so often, you ping them so they pick up your current IP address and the DNS domain name you've purchased from them will then resolve to that IP when anyone does a DNS look up on "ice-cream.is-really-cool.com" (hence using the short TTL).

It's a bit like with a Jamulus Directory, really...